templates/leaderboard/leaderboard.html

```diff
@@ -2,35 +2,44 @@
 
 {% load staticfiles %}
 {% load humanize %}
+{% load perdiem %}
 
 {% block extrastyle %}
     <link rel="stylesheet" href="{% static "css/extra/stats.css" %}" type="text/css" charset="utf-8" />
 {% endblock %}
 
 {% block content %}
-    <h1>Investments</h1>
+    <h2>Investments</h2>
     <div class="break">
         <hr>
     </div>
+    <h2 class="green-color" id="average-return"></h2>
+    <h4>Overall Return</h4>
+    <div class="break">
+        <hr>
+    </div>
+    <h3>Investments</h3>
     <div class="center">
         {% include "leaderboard/includes/list.html" with name='Artists' leaders=top_raised %}
         {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_invested %}
     </div>
-    <div class="total-box">
-        <h1 class="white-color">${{ total_raised|floatformat:0|intcomma }}</h1>
-        <p class="white-color">AMOUNT OF MONEY RAISED FOR ARTISTS ON PERDIEM</p>
-    </div>
-    <br/>
-    <h1>Returns</h1>
     <div class="break">
         <hr>
     </div>
+    <h3>Earnings</h3>
     <div class="center">
         {% include "leaderboard/includes/list.html" with name='Artists' leaders=top_earned_artists %}
         {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_earned_investors %}
     </div>
-    <div class="total-box">
-        <h1 class="white-color">${{ total_generated|floatformat:0|intcomma }}</h1>
-        <p class="white-color">AMOUNT OF REVENUE GENERATED FOR ARTISTS AND INVESTORS</p>
-    </div>
+{% endblock %}
+
+{% block extrajs %}
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.1.4/Chart.min.js"></script>
```

Comment:
They updated to a new version of charts.js.

I can download it and replace all of the links if that would be better